Update documentation to reflect new repository location

The repository has been moved from CorfuDB/CorfuDB to CorfuDB/Corfu-Repo as a result of #257.

All documentation needs to be updated to reflect this new path.

Actually, I am not sure the repo is working, at least not with the instructions under Corfu-Repos README.

Here is a dump of the error you get from maven:

        INFO] Building CorfuDB-tutorials 1.0-SNAPSHOT
        [INFO] ------------------------------------------------------------------------
        Downloading:
        https://raw.github.com/corfudb/corfudb/mvn-repo/org/corfudb/corfu/0.1-SNAPSHOT/maven-metadata.xml
        Downloading:
        https://raw.github.com/corfudb/corfudb/mvn-repo/org/corfudb/corfu/0.1-SNAPSHOT/corfu-0.1-SNAPSHOT.pom
        [WARNING] The POM for org.corfudb:corfu:jar:0.1-SNAPSHOT is missing, no
        dependency information available
        Downloading:
        https://raw.github.com/corfudb/corfudb/mvn-repo/org/corfudb/corfu/0.1-SNAPSHOT/corfu-0.1-SNAPSHOT.jar
        [INFO] ------------------------------------------------------------------------
        [INFO] BUILD FAILURE
        [INFO] ------------------------------------------------------------------------
        [INFO] Total time: 3.256 s
        [INFO] Finished at: 2016-11-19T10:52:45-08:00
        [INFO] Final Memory: 9M/155M
        [INFO] ------------------------------------------------------------------------
        [ERROR] Failed to execute goal on project CorfuDB-tutorials: Could not resolve
        dependencies for project
        org.corfudb.tutorials:CorfuDB-tutorials:jar:1.0-SNAPSHOT: Could not find
        artifact org.corfudb:corfu:jar:0.1-SNAPSHOT in corfu-github-mvn-repo
        (https://raw.github.com/corfudb/corfudb/mvn-repo/) -> [Help 1]
        org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal
        on project CorfuDB-tutorials: Could not resolve dependencies for project
        org.corfudb.tutorials:CorfuDB-tutorials:jar:1.0-SNAPSHOT: Could not find
        artifact org.corfudb:corfu:jar:0.1-SNAPSHOT in corfu-github-mvn-repo
        (https://raw.github.com/corfudb/corfudb/mvn-repo/)

--- a/.travis_scripts/mvnrepo.sh
+++ b/.travis_scripts/mvnrepo.sh
@@ -7,12 +7,8 @@ if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; th
         PROJECT_VERSION=$(cat ${HOME}/.project_version)
         mvn -N io.takari:maven:wrapper -Dmaven=3.3.9
         ./mvnw deploy -DskipTests=true
-        #delete debian artifacts before deploy
-        rm target/mvn-repo/org/corfudb/corfu/${PROJECT_VERSION}/*.deb
-        rm target/mvn-repo/org/corfudb/corfu/${PROJECT_VERSION}/*.deb.md5
-        rm target/mvn-repo/org/corfudb/corfu/${PROJECT_VERSION}/*.deb.sha1
         cp -R target/mvn-repo $HOME/mvn-repo-current
-        cp -R runtime/target/mvn-repo $HOME/mvn-repo-current
+        cp -R runtime/target/mvn-repo/* $HOME/mvn-repo-current
         cd $HOME
         git config --global user.email "travis@travis-ci.org"
         git config --global user.name "travis-ci"

